content/doc/book/security/pipeline-authors.adoc

```diff
@@ -0,0 +1,33 @@
+---
+title: Security Practices for Pipeline Authors
+layout: section
+---
+ifdef::backend-html5[]
+:toc:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+endif::[]
+
+The job of securing the Jenkins instance falls mostly on administrators but Pipeline authors must also adhere to good security practices.
+We summarize these here.
+
+== Use Credentials to Access Resources
+
+If your Pipeline needs to access external resources such as a database, artifact repository, or cloud, be sure to use credentials [add link] for authorization rather than coding the username/password, secret text, or other identifiers in your Pipeline.
```

Comment:
```suggestion
If your Pipeline needs to access external resources such as a database, artifact repository, or cloud, be sure to use credentials [add link] for authorization rather than hardcoding the username/password, secret text, or other identifiers in your Pipeline.
```

Not sure the simple "coding" is good in this context. (Lacking english expertize)

Reply:
Your suggestion is a definite improvement -- thanks!